CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (store) [#14410](https://github.com/cosmos/cosmos-sdk/pull/14410) `rootmulti.Store.loadVersion` has validation to check if all the module stores' height is correct, it will error if any module store has incorrect height.
 * (store) [#14189](https://github.com/cosmos/cosmos-sdk/pull/14189) Add config `iavl-lazy-loading` to enable lazy loading of iavl store, to improve start up time of archive nodes, add method `SetLazyLoading` to `CommitMultiStore` interface.
```

Comment:
Do we need a remark about what to add in the app.toml like in v0.46?

I still need to focus back on https://github.com/cosmos/cosmos-sdk/issues/14630 so it won't be as important anymore, but https://github.com/cosmos/cosmos-sdk/blob/main/tools/confix/data/ should be updated with the new field as well.

Reply:
what do you mean by updated?

Reply:
Explained here: https://github.com/cosmos/cosmos-sdk/pull/14876